PraceDomowe/PracaDomowa5/Drzewiecki_Marcin_PD5.ipynb

```diff
@@ -0,0 +1,189 @@
+{
```

Comment:
<p>Nie o to chodziło, <code>PostId</code>  to numer identyfikujący post, nie jest on związany z liczbą komentarzy związanych z danym postem. Metoda rysowania histogramu została użyta poprawnie.</p><p> </p><p><code>0.5 punktu</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>